core/serialisation/src/test/java/uk/gov/gchq/gaffer/serialisation/implementation/tostring/StringToStringSerialiserTest.java

```diff
@@ -40,15 +37,16 @@ public void shouldSerialiseAndDeserialise() throws Exception {
         assertEquals(serialised, deserialise);
     }
 
-
+    @Test
```

Comment:
Hi - in the JUnit5 Jupiter docs for @Test annotation, the docs say 
> "Denotes that a method is a test method. Unlike JUnit 4’s @Test annotation, this annotation does not declare any attributes, since test extensions in JUnit Jupiter operate based on their own dedicated annotations. Such methods are inherited unless they are overridden."

Source: https://junit.org/junit5/docs/current/user-guide/#writing-tests-annotations

So that is the reason for adding it so that it is recognised as a test. 

One way I tested this was to add an Assertions._fail()_ in an @Override inherited test with no annotation and the build still passed - when I added the @Test (with the _fail()_ sill in the test) it ran the test and failed the build.

Reply:
I'll resolve this comment and keep the initial @Test @Override comment open above